slf4j/src/main/scala-2/org/typelevel/log4cats/slf4j/Slf4jLogger.scala

```diff
@@ -18,13 +18,19 @@ package org.typelevel.log4cats.slf4j
 
 import cats.effect.Sync
 import org.typelevel.log4cats.SelfAwareStructuredLogger
-import org.typelevel.log4cats.slf4j.internal._
+import org.typelevel.log4cats.slf4j.internal.{_}
 import org.slf4j.{Logger => JLogger}
+import scala.annotation.nowarn
 
 object Slf4jLogger {
 
-  def getLogger[F[_]: Sync]: SelfAwareStructuredLogger[F] =
-    macro GetLoggerMacros.unsafeCreateImpl[F[_]]
+  @nowarn("cat=unused")
+  private def getLogger[F[_]: Sync]: SelfAwareStructuredLogger[F] = ???
+  @nowarn("cat=unused")
+  private def create[F[_]: Sync]: F[SelfAwareStructuredLogger[F]] = ???
+
+  def getLogger[F[_]: Sync](implicit name: LoggerName): SelfAwareStructuredLogger[F] =
+    getLoggerFromName(name.value.stripSuffix("$"))
```

Comment:
It seems like the suffix stripping part is the `LoggerName` responsibility. I can't imagine any cases where anyone would need `FooService$`